docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md

```diff
@@ -0,0 +1,85 @@
+
+You’ve just created your enrollment group. The generated Symmetric Key will be used in the next step.
+
+!!! info
+    On the Enrollment Group Details page you can also select to which hub the enrollment group will be assigned. The list of hubs that are connected to this DPS is displayed in the drop-down menu under **Select the IoT hubs this group can be assigned to**. To add more hubs to this list, click **Link a new IoT hub** or go to the Linked IoT hubs page from the left pane.
+
+## Get the credentials
+
+This step explains where you can get the credentials for setting up the Azure DPS extension in Coiote DM.
+
+### ID Scope
+
+The ID Scope is used to identify the specific provisioning service for the device registration. The ID Scope is automatically generated and unique.
+
+1. Log in to the [Azure portal](https://portal.azure.com) and click on your **Device Provisioning Service**.
+2. On the Overview page, locate the **ID Scope** under the Essentials section. Copy it into Notepad or another place to keep it for later.
+
+![Copy the ID Scope from the Overview panel](images/dps-4.png "Copy the ID Scope from the Overview panel")
+
+### Enrollment group key
+
+1. Select **Manage enrollments** from the left pane of your Azure DPS.
+2. Under the **Enrollment Groups** tab, click on the group that you want to use.
+
+    ![Click on the Enrollment group you want to use](images/dps-5.png "Click on the Enrollment group you want to use")
+
+3. In the **Settings** tab, under Attestation Type, copy **Primary Key.**
+
+![Copy the Primary key](images/dps-6.png "Copy the Primary key")
+
+## Set up the Azure DPS extension
+
+1. In your Coiote DM account, go to **Administration —> Hyperscaler Integration Center**.
+2. In the **Integration** tab, locate the Azure DPS tale and click **Connect**.
+
+    ![Click on the Azure DPS tab](images/dps-7.png "Click on the Azure DPS tab")
+
+3. In the dialog that appears, provide the credentials for your Azure DPS integration:
+    * **ID Scope**: paste the ID Scope copied from the Overview page.
+    * **Enrollment group key:** paste the Primary Key copied from the Enrollment group.
+    * **Device Provisioning Service hostname** can be left as it is: `global.azure-devices-provisioning.net`
+
+![Enter ID Scope, Enrollment group key and click Save](images/dps-8.png "Enter ID Scope, Enrollment group key and click Save")
+
+## What happens next
+
+After you set up Azure DPS and configure your Azure DPS extension in Coiote DM, device provisioning and reprovisioning are automated. You aren’t required to make any changes in Coiote DM.
+
+Importing and exporting devices, upgrading device firmware, and setting observations — all these operations happen at the level of individual hubs. The detailed instructions are provided in our [Azure IoT Hub documentation](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Device_operations/Overview/).
```

Comment:
I'm not sure that this part is clear. I understand that, but I guess that may be messy for people who do not have enough understanding and knowledge about that. When the DPS extension is configured all devices are provisioned to the proper IoT Hub using that DPS service. Linking IoT hub documentation may suggest that the user is required to add IoT hub somehow. 

But this is just my reflection.

Reply:
I'm not sure now how to put it in another way, but will think about it.